code/modules/antagonists/cult/cult_comms.dm

```diff
@@ -135,7 +135,7 @@
 	for(var/datum/mind/B in team.members)
 		if(B.current)
 			for(var/datum/action/innate/cult/mastervote/vote in B.current.actions)
-				vote.Remove(B.current)
+				qdel(vote)
```

Comment:
Shouldn't you also remove it from the list to remove the ref?

Reply:
`/datum/action/Destroy` calls `Remove(owner)` which removes itself from `actions` list of the owner:

https://github.com/BeeStation/BeeStation-Hornet/blob/026656b20ba1c250f0975c8d260ff003480d640e/code/datums/action.dm#L80-L89

This behavior has been the reason why various cleanup code has been calling only `Remove`(not recommended) or `qdel`(recommended) and be sorta okay.

Reply:
👌🏻